### ISSUE TITLE
bump scikit-learn version ceiling to 0.20.1?

### ISSUE THREAD

**Issue by bdewilde** [dragnet-org/dragnet#76]
Hi! `dragnet` currently requires `scikit-learn<=0.19.2`, which prevents users from developing alongside the final Py2/3 compatible release of 0.20.1 (see [here](http://scikit-learn.org/dev/whats_new.html#version-0-20-0)). Would you be able to bump the version ceiling and publish a new release? Assuming no version-driven incompatibilities, I _think_ this shouldn't be too much work; you don't have to train new, version-specific models, right?

**Comment by pakelley** [comment]
I don't expect any issues with that, and tests are passing on Py2/3 with scikit-learn 0.20.0.
I'll poke around and spot check things sometime in the next week or so on 0.20.0 just to be sure, but that seems like an easy change (and yeah, there shouldn't need to be any new models trained).

**Comment by pakelley** [comment]
Alright, I got a little busy, but I finally had the chance to look at this again today. Looks like the accuracy and throughput for both pre-trained and newly-trained models are unaffected by using sklearn 0.20. Opening a PR to bump the ceiling now
